core/kazoo_provisioner/src/provisioner_v5.hrl

```diff
@@ -4,6 +4,7 @@
 -include_lib("kazoo_stdlib/include/kz_databases.hrl").
 
 -define(MOD_CONFIG_CAT, <<"provisioner">>).
+-define(MOD_CONFIG_CLUSTER, <<"cluster">>).
```

Comment:
delete this

Reply:
Where should provisioner_v5.erl reference that from then?

Reply:
include `crossbar.hrl`

Reply:
a better way would be to create an accessor ( `kzd_cluster` ) in kazoo_documents and use it

Reply:
it would also be used in the webhooks pr